cg/meta/workflow/rnafusion.py

```diff
@@ -93,6 +89,19 @@ def build_samplesheet_content(
         }
         return samplesheet_content
 
+    def create_samplesheet_csv(
```

Comment:
```suggestion
    def write_samplesheet_csv(
```
This function seems to not create but write the sample sheet